Documentation/ExtensionArchitecture/FileStructure/Resources/Private/Index.rst

```diff
@@ -3,27 +3,149 @@
+
+    Folder `Templates` often contains the Fluid templates for a TYPO3 extension.
+
+..  typo3:file:: SomePartials.html
+    :scope: extension
+    :path: /Resources/Private/Partials/
+    :regex: /^.*\/Resources\/Private\/Partials\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Partials for an extension
+
+    Folder `Partials` often contains the Fluid partials for a TYPO3 extension.
+    These can be included via the `Render ViewHelper <f:render> <https://docs.typo3.org/permalink/t3viewhelper:typo3-fluid-render>`_
+    into the main Fluid template.
+
+..  typo3:file:: SomeLayout.html
+    :scope: extension
+    :path: /Resources/Private/Layouts/
+    :regex: /^.*\/Resources\/Private\/Layouts\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Layouts for an extension
+
+    Folder `Layouts` often contains the Fluid layouts for a TYPO3 extension.
+    These can be included via the `Layout ViewHelper <f:layout> <https://docs.typo3.org/permalink/t3viewhelper:typo3fluid-fluid-layout>`_
+    into the main Fluid template.
+
+.. _extension-resources-private-fluid-site-packages:
+
+Common Fluid template locations for site packages
+=================================================
+
+Commonly site package in TYPO3 v13 and above use the `PAGEVIEW <https://docs.typo3.org/permalink/t3tsref:cobj-pageview>`_
+TypoScript object to display the HTMl page output. They have one folder, commonly
+`PageView` or `Templates` in folder `Resources/Private` with the subfolders
+`Pages`, `Partials` and `Layouts` (they cannot be renamed).
+
+..  typo3:file:: MyPageLayout.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Pages/
+    :regex: /^.*\/Resources\/Private\/PageView\/Pages\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Templates for different page layouts
+
+    This folder contains one Fluid template for each page layout defined in the
+    site package. See
+
+..  typo3:file:: SomePartials.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Partials/
+    :regex: /^.*\/Resources\/Private\/PageView\/Partials\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Partials for the page view
+
+    Folder `Partials` often contains the Fluid partials for a TYPO3 extension.
+    These can be included via the `Render ViewHelper <f:render> <https://docs.typo3.org/permalink/t3viewhelper:typo3-fluid-render>`_
+    into the main Fluid template.
+
+..  typo3:file:: SomeLayout.html
+    :scope: extension
+    :path: /Resources/Private/PageView/Layouts/
+    :regex: /^.*\/Resources\/Private\/PageView\/Layouts\/[A-Za-z0-9]+\.html$/
+    :shortDescription: Fluid Layouts for the page view
+
+    Folder `Layouts` often contains the Fluid layouts for a TYPO3 extension.
+    These can be included via the `Layout ViewHelper <f:layout> <https://docs.typo3.org/permalink/t3viewhelper:typo3fluid-fluid-layout>`_
+    into the main Fluid template.
+
+Templates to override or extend Fluid-Styled Content based content objects are
```

Comment:
Maybe reference `styles.templates.templateRootPath` here again to make clear that this needs to be configured first?